### ISSUE TITLE
TypeError: Expected int32, got list containing Tensors of type '_Message' instead.

### ISSUE THREAD

**Issue by hnsywangxin** [tmulc18/Inception#1]
when I runned the code ,I got a problem in  the line  inception1 = tf.nn.relu(tf.concat(3,[conv1_1x1_1,conv1_3x3,conv1_5x5,conv1_1x1_4]))

my python version is 2.7
pandas version is  0.19.2 
numpy version is 1.12.1
tensorflow version is 1.0.1
my OS is ubuntu 16.04
why?
thanks

why? thanks

**Comment by tmulc18** [comment]
The code posted on GitHub was written with an older version of TensorFlow.  Please see the documentation for you version of TensorFlow for the methods in that line.  For example, I believe in your version you will need to specify the values and axis for the concat function.  If you tell me your exact error, I might be able to be of more help.

**Comment by tmulc18** [comment]
After checking the documentation for your version of TensorFlow, it looks like in your method call you need to either specify the values and axis, or reverse their order.   Thank you for bringing up this issue!

**Comment by hnsywangxin** [comment]
I have solved it,thank you@tmulc18

**Comment by AbNeil** [comment]
Hello @tmulc18 
I tried to implement your inception module. I am facing an error
TypeError: unbound method as_default() must be called with Graph instance as first argument (got nothing instead)
However I solved this problem by using tf.Graph().as_default() instead
But now it gives me an error
TypeError: graph must be a tf.Graph, but got <type 'type'>
I am using python 3.6
Tensorflow 0.12.
Any help would be appreciated.
